<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>음악 플레이어</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="past_button">
       <property name="text">
        <string>이전 음악</string>
       </property>
       <property name="shortcut">
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause_button">
       <property name="text">
        <string>일시정지</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="text">
        <string>다음 음악</string>
       </property>
       <property name="shortcut">
        <string>Right</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="AddSong">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>420</y>
      <width>801</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>추가</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
